system-tests/lib/system-tests.ts

```diff
@@ -960,11 +972,17 @@ const systemTests = {
         } catch (err) {
           // firefox has issues with recording video. for now, ignore snapshot diffs that only differ in this error.
           // @see https://github.com/cypress-io/cypress/pull/16731
-          if (!(options.browser === 'firefox' && isVideoSnapshotError(err))) {
+          if (!(options.browser === 'firefox' && (isVideoSnapshotError(err) || isFirefoxConnectionError(err)))) {
             throw err
           }
 
-          console.log('(system tests warning) Firefox failed to process the video, but this is being ignored due to known issues with video processing in Firefox.')
+          if (isVideoSnapshotError(err)) {
+            console.log('(system tests warning) Firefox failed to process the video, but this is being ignored due to known issues with video processing in Firefox.')
+          }
+
+          if (isFirefoxConnectionError(err)) {
```

Comment:
instead of doing this here, can you do it in `normalizeStdout`? i.e. strip out the connection timeout lines. it's the same thing we do for normalizing all the other variable stdout.